cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -71,7 +71,7 @@ func GetKubeConfigPath(file string) string {
 	if file != "" {
 		return file
 	}
-	// Find a config in the standard locations using DefaultClientConfigLoadingRules,
+	// Find a config in the standard locations using NewDefaultClientConfigLoadingRules,
```

Comment:
DefaultClientConfigLoadingRules is a valid structure under client go. New* just creates it.

thanks, but this change is not really needed.
/close